src/main/scala/dissonance/model/package.scala

```diff
@@ -1,12 +1,18 @@
 package dissonance
 
+import io.circe.{Decoder, Encoder}
 import io.estatico.newtype.macros.newtype
 
 package object model {
   // TODO: Implement
   type Snowflake = Long
   // TODO: newtypes for all kinds of Snowflakes
   @newtype case class DiscordId(value: Snowflake) // TODO: Change to Snowflake
```

Comment:
Do we need this todo? Isn't it already using snowflakes?

Reply:
Yep. I'll remove it